files/nrpe/check_openstack.py

```diff
@@ -672,6 +681,53 @@ def execute(self):
     except:
       raise
 
+class OSL3Agent():
+  '''
+  Check all Neutron L3 agents are alive
+  '''
+
+  options = dict()
+
+  def __init__(self, options):
+    self.neutron = neutronclient.Client('2', session=keystone_session_v3(options))
+
+  def check_bad_l3_agents(self):
+    err_l3agents = []
```

Comment:
I would keep consistency if in line 689 you are using dict(), here you can use list(). For me calling list() is more readable, and it seems like [] might reuse memory cells https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better

Reply:
I prefer {}, [] style and also pylint advices that way: Consider using '{}' instead of a call to 'dict'. (use-dict-literal). But I have now changed it to list() so it's consistent with other code.